### ISSUE TITLE
share local screen over internet

### ISSUE THREAD

**Issue by cddharthsingh** [phoboslab/jsmpeg-vnc#72]
is there any way by which I can share my screen over the internet, not just in my local wifi network?

**Comment by Maneesh3** [comment]
Yes, port forward the assigned port to the application in the router. Also, you must have a public IP address to make this work.

**Comment by Maneesh3** [comment]
Duplicate of #32
